src/outlet_registry.js

```diff
@@ -59,4 +59,8 @@ OutletRegistry.prototype.update = function(outlet_id, component) {
     }
 };
 
+OutletRegistry.prototype.rewind = function(){
```

Comment:
I am going to rename this to `clear()`.  I think its more obvious than `rewind()`